Tests/Form/Type/SimpleMoneyTypeTest.php

```diff
@@ -72,12 +62,33 @@ public function testGreaterThan1000Valid()
     public function testSetData()
     {
         \Locale::setDefault("fr_FR");
-        $moneyType = new SimpleMoneyType($this->pairManager, 2);
-        $form = $this->factory->create($moneyType, null, array());
+        $form = $this->factory->create($this->simpleMoneyTypeClass, null, array());
         $form->setData(Money::EUR(120));
         $formView = $form->createView();
 
         $this->assertEquals("1,20", $formView->children["tbbc_amount"]->vars["value"]);
     }
 
+    protected function getExtensions()
+    {
+        //This is probably not ideal, but I'm not sure how to set up the pair manager
+        // with different decimals for different tests in Symfony 3.0
```

Comment:
I tried this example in CurrencyTypeTest and removed getExtensions(), but got an error message saying that I was missing the arguments for the constructor of the CurrencyType .  This was on Symfony 3.0.  Looking through the code in FormRegistry, I don't see how the options as you described would ever get to the constructor, so while I agree that we should initialize objects for each test separately, I don't see how this would work or how it would work in Symfony 3.0 at this point.  Here's a thread I had found that inspired how it's structured right now: https://github.com/symfony/symfony-docs/issues/6053